### PR TITLE
Add a dvd screensaver that changes colors.

### PR DESCRIPTION
**Pull request by agne6ka**
Exercise 1) 
A DVD screensaver first attempt.

### CODE REVIEW

File: SDL2Game/main.cpp [hunk 1]
```diff
@@ -1,20 +1,32 @@
...
+#include <random>
 
-const int WIDTH = 800, HEIGHT = 600;
+const int SCREEN_WIDTH = 800, SCREEN_HEIGHT = 600;
 
 // TODO: exercise 1) dvd screensaver, bouncing rect & change colors
 // TODO: exercise 2) dvd screensaver, add texture to rect
+int randomize(int maxRange)
+{
+    std::random_device rd;                              // obtain a random number from hardware
```
Comment (ragnarlodbrok1992) [inline SDL2Game/main.cpp]: This random devices gets initiated every time we call for random value. Maybe try to move it to some static members and just "get" random value from it, do not initialize it every time.

**Review by ragnarlodbrok1992 — CHANGES_REQUESTED** [review]
Comment about random_device.

File: SDL2Game/main.cpp (hunk 1 above)
Comment (agne6ka) [reply SDL2Game/main.cpp]: I'll move it to the utils, does that sound good 🤔? I was also thinking about whether it is better to use an approach with modulo.

Comment (agne6ka) [reply SDL2Game/main.cpp]: https://learn.microsoft.com/en-us/shows/goingnative-2013/rand-considered-harmful 
Heh now I know the answer 😄

**Review by ragnarlodbrok1992 — APPROVED** [review]
Looks awesome :D

**Pull request merged**